src/entities/User/model/selectors/index.ts

```diff
@@ -0,0 +1,10 @@
+import { StateSchema } from "app/providers/StoreProvider/config/schema";
+
+
+export const getUser = (state: StateSchema) => ({
+    id: state.user.id,
+    email: state.user.email,
+    token: state.user.token,
+})
+
+export const isAuthorized = (state: StateSchema) => Boolean(state.user.id);
```

Comment:
use prefix get at selector
```suggestion
export const getIsAuthorized = (state: StateSchema): boolean => Boolean(state.user.id);
```